UnityProject/Assets/Tests/ExplodeWhenShotTest.cs

```diff
@@ -51,7 +51,7 @@ public IEnumerator Should_Destroy_Bullet()
 	[UnityTest]
```

Comment:
We haven't enforced Unit Testing for awhile (as it became a hassle to maintain with the rapid changes) but thanks for adjusting this also.